lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -239,6 +244,10 @@ def check_mode?
         @check_mode == true && @rex_command == false
       end
 
+      def diff_mode?
+        @diff_mode.nil? ? false : @diff_mode == true
```

Comment:
Wouldn't this do exactly the same thing?
```suggestion
        @diff_mode == true
```

Reply:
Yes, of course :-)